locations/spiders/orlen.py

```diff
@@ -13,7 +13,7 @@ class OrlenSpider(scrapy.Spider):
     name = "orlen"
     custom_settings = {"ROBOTSTXT_OBEY": False}
     brand_mappings = {
-        "ORLEN": {"brand": "Orlen", "brand_wikidata": "Q971649"},
+        "ORLEN": {"brand": "Orlen Paczka", "brand_wikidata": "Q110457879"},  # ?
```

Comment:
Orlen Paczka is for parcel lockers (though they also should rather use `brand:wikidata` pointing to Orlen brand)

Orlen is for fuel stations.